recipes/libsass/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "libsass" %}
+{% set version = "0.13.2" %}
+{% set sha256 = "f6270e6f5eb0bfa4e5ecf5b9e6443f576a6fe47cbb732b8d6acaf396c323ec0a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true # [win and py2k]
```

Comment:
Apparently github collapses multiple spaces in code formatting somewhere! It should be `skip:_true__#_[win_and_py2k]`, with each `_` replaced by a space.